packages/core/FieldColumn.h

```diff
@@ -282,6 +282,7 @@ FieldColumn<T>& FieldColumn<T>::operator= (const FieldColumn<T>& column)
     }
 
     encoding = column.encoding;
+    chunkSize = column.chunkSize;
```

Comment:
This is a bug.  The chunkSize has to be the same as was used in the lines above when appending elements.